modules/ROOT/pages/import-and-export-preferences-studio.adoc

```diff
@@ -0,0 +1,37 @@
+= Export and Import Studio Preferences
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can export your Anypoint Studio preferences and import them into a new version.
+
+Follow this process if you had to download a fresh copy of Studio and you want to recover your previous configurations.
+
+== Export Your Studio Configurations
+
+. In Studio, go to *Preferences*.
+. Ensure that you applied all the configurations that you want to export.
+. Select the *Export* icon image:export-icon.png[30,30]
```

Comment:
Is the icon * labeled* "Export"? If not, suggest "Select the [image] icon:"

Reply:
Or "Select the export icon:" and let the image illustrate what that type of icon looks like, since it is not labeled.